src/Proxy.NET/Models/Deployment.cs

```diff
@@ -0,0 +1,11 @@
+namespace Proxy.NET.Models;
+
+public partial class Deployment
```

Comment:
Should this be in the Database folder/namespace? Looks to be used by the DbContext.

Reply:
Punting to [#307](https://github.com/microsoft/azure-openai-service-proxy/issues/307) and #308